cookbook/integrations/kubernetes/pod/pod.py

```diff
@@ -49,65 +31,97 @@
     V1VolumeMount,
 )
 
-_SHARED_DATA_PATH = "/data/message.txt"
-
 # %%
-# We define a simple pod spec with two containers.
-
+# Simple example
```

Comment:
Maybe we call this - using pod spec to add additional properties to the launched task container 

This could be using node selectors, affinity etc.

Also we should reference pod template config for system wide defaults. Cc @hamersaw

Reply:
Are you referring to https://github.com/flyteorg/flyte/issues/3123?

Reply:
@samhita-alla I assume this is referring to the [default PodTemplate](https://docs.flyte.org/en/latest/deployment/cluster_config/general.html#using-default-k8s-podtemplates). I think this example is pushing the PodTask towards the proposal you linked above; however, it doesn't apply to ContainerTask or python tasks like adding a `PodTemplate` to the task decorator could.

Reply:
@hamersaw, python or pytorch?

Reply:
So, I think this PR is great! In regards to Ketans comment I think just a little note describing the default PodTemplate work as configuring k8s resources system-wide, rather than on a specific task is enough. The goal is for the [default PodTemplate](https://docs.flyte.org/en/latest/deployment/cluster_config/general.html#using-default-k8s-podtemplates) to be applied to every k8s Pod Flyte creates - so it will apply to PodTasks, ContainerTasks, PyTorch, Spark, Ray, Dask, etc. In saying "python task" I just meant the base Flyte task, ie:
```
@task
def hello_world(name: str) -> str:
    return f"{hello {name}"
```

Reply:
Done!